### PR TITLE
add MediaConnect and MediaLive metrics

### PR DESCRIPTION
**Pull request by copperlight**

### CODE REVIEW

File: atlas-cloudwatch/src/main/resources/mediaconnect.conf [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+          tags = [
+            {
+              key = "id"
+              value = "overflow"
+            }
+          ]
+        },
+        {
+          name = "SourceMissingPackets"
+          alias = "aws.mediaconnect.packets"
```
Comment (brharrington) [inline atlas-cloudwatch/src/main/resources/mediaconnect.conf:143]: Should this be `sourcePackets` to be consistent with the errors?

File: atlas-cloudwatch/src/main/resources/medialive.conf [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+          tags = [
+            {
+              key = "id"
+              value = "max"
+            }
+          ]
+        },
+        {
+          name = "NetworkIn"
+          alias = "aws.medialive.NetworkThroughput"
```
Comment (brharrington) [inline atlas-cloudwatch/src/main/resources/medialive.conf:49]: `networkThroughput` to be consistent with other names

File: atlas-cloudwatch/src/main/resources/medialive.conf [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+          tags = [
+            {
+              key = "id"
+              value = "in"
+            }
+          ]
+        },
+        {
+          name = "NetworkOut"
+          alias = "aws.medialive.NetworkThroughput"
```
Comment (brharrington) [inline atlas-cloudwatch/src/main/resources/medialive.conf:60]: `networkThroughput`

**Review by brharrington — APPROVED** [review]
Mostly looks fine, a few minor suggestions.

**Comment by copperlight** [comment]
Updates applied for suggestions, will hold on merging this until the documentation PR is submitted.

**Pull request merged**